pkg/datarepair/repairer/repairer.go

```diff
@@ -7,15 +7,30 @@ import (
 	"context"
 	"time"
 
+	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 
+	"storj.io/storj/internal/memory"
 	"storj.io/storj/internal/sync2"
 	"storj.io/storj/pkg/datarepair/queue"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pointerdb"
+	ecclient "storj.io/storj/pkg/storage/ec"
+	"storj.io/storj/pkg/storage/segments"
 	"storj.io/storj/pkg/storj"
 	"storj.io/storj/pkg/transport"
 	"storj.io/storj/storage"
 )
 
+// Config contains configurable values for repairer
+type Config struct {
+	MaxRepair     int           `help:"maximum segments that can be repaired concurrently" default:"100"`
+	Interval      time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	OverlayAddr   string        `help:"Address to contact overlay server through"`
```

Comment:
This shouldn't be necessary anymore.

Reply:
It is used in `repairer.Run`: `oc, err = overlay.NewClientContext(ctx, service.transport, service.config.OverlayAddr)`
Should this be changed?

Reply:
Both are internal services for satellite so it makes sense to call them directly rather than going through costly grpc.

Reply:
Is the same true of ecclient?

Reply:
Also, there is no overlay.Service as far as I know. Should I use overlay.Server?

Reply:
`ecclient` is necessary because storage nodes aren't internal to the satellite.

Nothing should directly depend on a server nor endpoint (with the only exception being peer, which needs to wire them up). This is what got into the "APIKey" mess in the first place.

But you could use `overlay.Cache`

Reply:
PS: you can do this in a separate PR, if it makes life easier.

Reply:
I'll make a separate PR for it.